<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>c_antinet_agent_api</class>
 <widget class="QMainWindow" name="c_antinet_agent_api">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>c_antinet_agent_api</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Emit tokens</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>All emited tokens</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="emitValue"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="emitToken"/>
       </item>
       <item row="1" column="2" rowspan="2">
        <widget class="QListWidget" name="lw_emited"/>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="buttonEmit">
         <property name="text">
          <string>Emit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Send</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignVCenter</set>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" rowspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>for</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4" rowspan="5">
        <widget class="QListWidget" name="lw_wallet"/>
       </item>
       <item row="3" column="3">
        <widget class="QComboBox" name="receivedTokens"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="receivedValue"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>to</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>of</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QLineEdit" name="sendValue"/>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Your wallet</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>of</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="2">
        <widget class="QComboBox" name="sendToken"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="receiver"/>
       </item>
       <item row="4" column="1" rowspan="2" colspan="3">
        <widget class="QPushButton" name="buttonXfer">
         <property name="text">
          <string>Xfer</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Logs</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="te_msgView"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>539</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
